### ISSUE TITLE
[expo-optimize]: Cannot find module 'glob'

### ISSUE THREAD

**Issue by jaulz** [expo/expo-cli#2188]
## Description

Running `npx expo-optimize` fails since the last release.

## Expected Behavior

Module should be executed without errors.

## Observed Behavior

```
Cannot find module 'glob'
Require stack:
- /Users/Julian/.npm/_npx/9050/lib/node_modules/expo-optimize/node_modules/@expo/config/build/ios/BundleIdentifier.js
- /Users/Julian/.npm/_npx/9050/lib/node_modules/expo-optimize/node_modules/@expo/config/build/ios/index.js
- /Users/Julian/.npm/_npx/9050/lib/node_modules/expo-optimize/node_modules/@expo/config/build/index.js
- /Users/Julian/.npm/_npx/9050/lib/node_modules/expo-optimize/build/index.js
```

## Environment

n/a

## Reproducible Demo

Just run `npx expo-optimize`.

**Comment by fson** [comment]
Thanks for reporting! Fixed in the latest `expo-optimize` (0.1.31).

**Comment by jaulz** [comment]
That was quick. Thanks!
